### PR TITLE
Refactor UI

### LINKED ISSUE #7: refactor-ui
Refactored the Code
Shifted to UseReducer and made all the API calls in utils 
Changed The UI for Notes and

### PR DESCRIPTION
**Pull request by sadathshariff**
Refactored the Code
Shifted to UseReducer and made all the API calls in utils 
Changed The UI for Notes and 

closes #7

### CODE REVIEW

File: src/components/AddNote/AddNote.jsx [hunk 1]
```diff
@@ -1,57 +1,56 @@
...
+  // const {
+  //   setUserNotes,
+  //   showForm,
+  //   setShowForm,
+  //   note,
+  //   setNote,
+  //   updateNote,
+  //   isEditing,
+  //   setIsEditing,
+  // } = useNotes();
```
Comment (rdshinde) [inline src/components/AddNote/AddNote.jsx]: Remove this if not needed.

File: src/context/Note/context.js [hunk 2]
```diff
@@ -1,67 +1,79 @@
...
+  //     });
+  //     const findDeletedNote = userNotes.find((item) => item._id === notesId);
+  //     if (res.status === 200) {
+  //       setUserNotes(res.data.notes);
+  //       setDeletedNotes([findDeletedNote, ...deletedNotes]);
+  //     }
+  //   } catch (error) {
+  //     console.error("Post Notes Error", error);
+  //   }
+  // };
```
Comment (rdshinde) [inline src/context/Note/context.js]: Here also, remove this if not needed

**Review by rdshinde — APPROVED** [review]
Looks good to merge. Just remove the commented code which is not required.

**Review by nishasen — APPROVED** [review]
Liked your editor code, learnt a lot from it.

**Pull request merged**